manifest.php

```diff
@@ -30,14 +31,20 @@
     'label' => 'Tao Sync Central Server',
     'description' => 'TAO central server for synchronisation.',
     'license' => 'GPL-2.0',
-    'version' => '0.1.0',
+    'version' => '0.2.0',
     'author' => 'Open Assessment Technologies SA',
-    'requires' => [],
+    'requires' => array(
+        'taoSync' => '>=7.1.0',
+    ),
     'managementRole' => SyncService::TAO_SYNC_ROLE,
     'acl' => [
         ['grant', SyncService::TAO_SYNC_ROLE, ['ext' => 'taoSyncServer']],
     ],
-    'install' => [],
+    'install'        => [
+        'php' => [
+            RegisterServices::class,
```

Comment:
It's better to use more specific names for install scripts to make it clear which services we are registering here.

Reply:
renamed to RegisterDataProviders